test/gerkin/tests/features/data_list.feature

```diff
@@ -0,0 +1,18 @@
+Feature: Searches and returns multiple records as an array-list.
+  Request endpoint: GET /data/{code}/{version}/
+
+  Background:
+    Given database with the valid schema has been set up
+
+  Scenario: Successfully get multiple records as an array-list
+    When I make a GET request with a valid Description Information-Mediator-Client
+    Then I receive a HTTP 200 response
+    And I received the records as an array-list
+
+  Scenario: Successfully get all records with name "John"
+    When I make a GET request with a valid Description Information-Mediator-Client and search "John"
```

Comment:
Please add information specify how given field is used in search context. 
`When I request entries that have FirstName equal to 'John'`